CHANGELOG.md

```diff
@@ -28,6 +28,7 @@ BUG FIXES:
 * command/show: Fix an issue with show and aliased providers ([#23848](https://github.com/hashicorp/terraform/issues/23848))
 * core: Always clean up empty resources before empty modules ([#23822](https://github.com/hashicorp/terraform/issues/23822))
 * internal/modsdir/manifest: Fix CLI issue with Windows machines ([#23865](https://github.com/hashicorp/terraform/issues/23865))
+* backend: Variable values are no longer output to the console when variables are undeclared [[#22782](https://github.com/hashicorp/terraform/issues/22782)]
```

Comment:
Normally we have a bot that expands these links, but since you're modifying a released changelog you might not know that this should be a link to the PR #23293, not the closed issue:  
https://github.com/hashicorp/terraform/issues/23293

(gh redirects that to https://github.com/hashicorp/terraform/pull/23293)

Reply:
Being totally nit-picky here, also note that we conventionally use parentheses around these links, not square brackets. :upside_down_face: